[rospy.client][INFO] 2019-11-14 18:07:16,576: init_node, name[/edrone/spawn_edrone], pid[5188]
[xmlrpc][INFO] 2019-11-14 18:07:16,577: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-11-14 18:07:16,578: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:41793/]
[rospy.init][INFO] 2019-11-14 18:07:16,578: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:41793/]
[rospy.impl.masterslave][INFO] 2019-11-14 18:07:16,578: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:41793/
[rospy.registration][INFO] 2019-11-14 18:07:16,580: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2019-11-14 18:07:16,580: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2019-11-14 18:07:16,679: registered with master
[rospy.rosout][INFO] 2019-11-14 18:07:16,679: initializing /rosout core topic
[rospy.rosout][INFO] 2019-11-14 18:07:16,683: connected to core topic /rosout
[rospy.simtime][INFO] 2019-11-14 18:07:16,685: initializing /clock core topic
[rospy.simtime][INFO] 2019-11-14 18:07:16,692: connected to core topic /clock
[rosout][INFO] 2019-11-14 18:07:16,696: Loading model XML from ros parameter
[rosout][INFO] 2019-11-14 18:07:16,703: Waiting for service /gazebo/spawn_urdf_model
[rospy.internal][INFO] 2019-11-14 18:07:16,920: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2019-11-14 18:07:17,227: topic[/clock] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:42789/], count 0
[rosout][INFO] 2019-11-14 18:07:17,309: Calling service /gazebo/spawn_urdf_model
[rosout][INFO] 2019-11-14 18:07:17,715: Spawn status: SpawnModel: Successfully spawned entity
[rospy.core][INFO] 2019-11-14 18:07:17,716: signal_shutdown [atexit]
[rospy.internal][INFO] 2019-11-14 18:07:17,720: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2019-11-14 18:07:17,720: topic[/clock] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:42789/
[rospy.impl.masterslave][INFO] 2019-11-14 18:07:17,721: atexit
[rospy.internal][WARNING] 2019-11-14 18:07:17,721: Unknown error initiating TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:50993 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:42789/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

